components/org.wso2.carbon.identity.media.core/src/main/java/org/wso2/carbon/identity/media/core/internal/MediaServiceComponent.java

```diff
@@ -45,13 +46,23 @@
     @Activate
     protected void activate(ComponentContext componentContext) {
 
-        BundleContext bundleContext = componentContext.getBundleContext();
-        MediaServiceDataHolder.getInstance().setBundleContext(bundleContext);
-        StorageSystemFactory fileBasedStorageSystemFactory = new FileBasedStorageSystemFactory();
-        bundleContext.registerService(StorageSystemFactory.class.getName(), fileBasedStorageSystemFactory, null);
-        StorageSystemFactory dbBasedStorageSystemFactory = new DatabaseBasedStorageSystemFactory();
-        bundleContext.registerService(StorageSystemFactory.class.getName(), dbBasedStorageSystemFactory, null);
-        bundleContext.registerService(StorageSystemManager.class, new StorageSystemManager(), null);
+        try {
+            // Load configurations from media.properties file into memory.
+            StorageSystemUtil.loadMediaProperties();
+
+            BundleContext bundleContext = componentContext.getBundleContext();
+            MediaServiceDataHolder.getInstance().setBundleContext(bundleContext);
```

Comment:
why do we nede to store the bundle context?

Reply:
removed it